components/WeeklyCard/index.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+const StyledLi = styled.li`
```

Comment:
Could you explain to me why? To me it feels more natural using the same name for the item as i would do in html

Reply:
Short Answer: Because it's easier to read and nicer to look at, and it will help you to not trip yourself up naming stuff in the future, such as `StyledLi2`, `StyledLiButDifferent`, but instead it's `StyledEventList`, `StyledCardList`. You may even omit the Styled part of it, but it makes it somewhat easier to differentiate between Styled Components and React Components.

Yes, that's the short answer. 😂

Reply:
Alright i got it, thank you 😂😂